pom.xml

```diff
@@ -24,7 +24,7 @@
   <properties>
     <java.level>8</java.level>
     <jenkins.version>2.150.1</jenkins.version>
-    <jcasc.version>1.29</jcasc.version>
```

Comment:
perhaps best to let dependabot take care of the bump

Reply:
no point, the tests would fail